scripts/azure_tags_env_variables.py

```diff
@@ -0,0 +1,16 @@
+from subprocess import Popen, PIPE
+import json
+
+raw_response = Popen(["curl", "-H", "Metadata:true", "http://169.254.169.254/metadata/instance?api-version=2017-08-01"], stdout=PIPE)
+res = raw_response.stdout.read()
+
+res_json = json.loads(res)
+res_tags = res_json['compute']['tags']
+
+file_obj = open("AzureTagsEnvVariables", "w")
+for pair in res_tags.split(';'):
+  pair = pair.encode('ascii', 'ignore').upper()
+  key_value = pair.split(':')
```

Comment:
Maybe print out the tags for debugging via the logs?

Reply:
Not sure if this is what you mean, let me know if you want me to look into importing some kind of logger